modules/tensor_mechanics/src/materials/abaqus/AbaqusUMATStress.C

```diff
@@ -204,11 +218,36 @@ AbaqusUMATStress::computeQpStress()
   const Real * myDFGRD1 = &(FBar_fortran(0, 0));
   const Real * myDROT = &(DROT_fortran(0, 0));
 
+  // More local copies of materials so we can rotate
+  RankTwoTensor stress_old = _stress_old[_qp];
+  RankTwoTensor total_strain_old = _total_strain_old[_qp];
+  RankTwoTensor strain_increment = _strain_increment[_qp];
+
+  // check if we need to rotate to intermediate configuration
+  if (_orientation_flag)
+  {
+    // keep track of total rotation
+    _total_rotation[_qp] = _rotation_increment[_qp] * _total_rotation_old[_qp];
```

Comment:
wait... shouldn't we keep track of the total rotation regardless of whether we specify an initial orientation or not?

Reply:
If not using some intermediate configuration, then all you need is current and previous configurations, which are already stored.

Reply:
I am having, I believe, a similar thought. Can we have a constant rotation, defining the initial anisotropy, that gets composed with rotation increments? I think that'd be more consistent with the rest of the code in `AbaqusUMATStress`. I may be missing something.

Reply:
There are two distinct behaviors - this is to mimic what Abaqus does. I agree it's not ideal or intuitive.